main.js

```diff
@@ -55,11 +55,28 @@
         });
     }
 
+    //Displays and hides the sidebar for mobile
+    function displaySidebar() {
+      document.getElementById("sidebar").style = "display:block";
+      document.getElementById("mobileMenuIcon").style = "display:none";
+      document.getElementById("mobileHideMenu").style = "display:block";
+    }
+
+    function hideSidebar() {
+      document.getElementById("sidebar").style = "";
+      document.getElementById("mobileMenuIcon").style = "";
+      document.getElementById("mobileHideMenu").style = ""
+    }
+
     //Add listeners for radio buttons
     document.getElementById("radioDay").addEventListener("click", displayPastDay);
     document.getElementById("radioWeek").addEventListener("click", displayPastWeek);
     document.getElementById("radioMonth").addEventListener("click", displayPastMonth);
 
+    //Listeners for mobile menu
+    document.getElementById("mobileMenuIcon").addEventListener("click", displaySidebar);
+    document.getElementById("mobileHideMenu").addEventListener("click", hideSidebar);
```

Comment:
If mobile, should start hidden and click should display, otherwise should start shown and click should hide. Need to check for that and set all that up, depending.